Fix compilation warnings

Can we fix these compilation warnings?

```
../src/api/wayfire/signal-definitions.hpp: In constructor ‘constexpr wf::view_mapped_signal::view_mapped_signal()’:
../src/api/wayfire/signal-definitions.hpp:346:8: warning: ‘wf::view_mapped_signal::is_positioned’ is deprecated: Use startup-x and startup-y properties instead [-Wdeprecated-declarations]
  346 | struct view_mapped_signal
      |        ^~~~~~~~~~~~~~~~~~
../src/api/wayfire/signal-definitions.hpp:352:10: note: declared here
  352 |     bool is_positioned = false;
      |          ^~~~~~~~~~~~~
../src/api/wayfire/signal-definitions.hpp:346:8: warning: ‘wf::view_mapped_signal::is_positioned’ is deprecated: Use startup-x and startup-y properties instead [-Wdeprecated-declarations]
  346 | struct view_mapped_signal
      |        ^~~~~~~~~~~~~~~~~~
../src/api/wayfire/signal-definitions.hpp:352:10: note: declared here
  352 |     bool is_positioned = false;
      |          ^~~~~~~~~~~~~
../src/view/view-impl.cpp: In function ‘void wf::view_implementation::emit_view_map_signal(wayfire_view, bool)’:
../src/view/view-impl.cpp:22:28: note: synthesized method ‘constexpr wf::view_mapped_signal::view_mapped_signal()’ first required here
   22 |     wf::view_mapped_signal data;
      |                            ^~~~
../src/view/view-impl.cpp:24:10: warning: ‘wf::view_mapped_signal::is_positioned’ is deprecated: Use startup-x and startup-y properties instead [-Wdeprecated-declarations]
   24 |     data.is_positioned = has_position;
      |          ^~~~~~~~~~~~~
../src/api/wayfire/signal-definitions.hpp:352:10: note: declared here
  352 |     bool is_positioned = false;
      |          ^~~~~~~~~~~~~
../src/view/view-impl.cpp:24:10: warning: ‘wf::view_mapped_signal::is_positioned’ is deprecated: Use startup-x and startup-y properties instead [-Wdeprecated-declarations]
   24 |     data.is_positioned = has_position;
      |          ^~~~~~~~~~~~~
../src/api/wayfire/signal-definitions.hpp:352:10: note: declared here
  352 |     bool is_positioned = false;
      |          ^~~~~~~~~~~~~
../src/view/view-impl.cpp:24:10: warning: ‘wf::view_mapped_signal::is_positioned’ is deprecated: Use startup-x and startup-y properties instead [-Wdeprecated-declarations]
   24 |     data.is_positioned = has_position;
      |          ^~~~~~~~~~~~~
../src/api/wayfire/signal-definitions.hpp:352:10: note: declared here
  352 |     bool is_positioned = false;
      |          ^~~~~~~~~~~~~
```

--- a/src/view/view-impl.cpp
+++ b/src/view/view-impl.cpp
@@ -21,7 +21,6 @@ void wf::view_implementation::emit_view_map_signal(wayfire_view view, bool has_p
 {
     wf::view_mapped_signal data;
     data.view = view;
-    data.is_positioned = has_position;
 
     view->emit(&data);
     if (view->get_output())

